pkg/plugins/sdk/datasource.go

```diff
@@ -0,0 +1,227 @@
+package grafana
```

Comment:
I conferred with @marefr, we both think it makes more sense to name the package "sdk" to correspond to the containing directory. What do you think?